hooks/android/sound/utilities.js

```diff
@@ -0,0 +1,78 @@
+"use strict"
+
+var path = require("path");
+var fs = require("fs");
+
+var utils = require("./utils");
+
+var constants = {
+  platforms: "platforms",
+  android: {
+    platform: "android",
+    wwwFolder: "assets/www",
+    soundFileExtension: ".wav",
+    soundFileName: "push_sound.wav",
```

Comment:
this doesn't seem to be referenced in the code

Reply:
you're right, nice catch!